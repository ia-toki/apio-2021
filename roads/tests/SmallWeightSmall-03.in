c10234cabdfd6eae47773fb6f163e0350fc797e1
200
82 193 8
155 11 7
159 82 7
18 118 8
90 193 8
16 38 8
74 16 9
111 163 7
54 99 10
88 28 7
193 190 7
157 64 8
83 189 8
100 63 10
118 5 10
179 163 9
81 193 9
168 82 8
137 28 9
135 118 10
183 151 9
0 11 10
189 193 10
104 81 9
76 81 10
65 144 7
99 79 8
62 28 10
86 82 9
150 151 8
162 82 10
194 164 9
182 151 9
195 72 8
80 11 10
101 82 10
164 79 7
124 130 9
64 48 7
181 153 8
22 61 8
10 154 8
96 189 7
154 81 10
198 28 9
2 28 8
66 164 9
197 119 7
60 28 9
8 101 7
148 69 8
63 164 10
192 85 9
9 151 7
38 80 9
89 70 10
7 189 7
153 11 10
105 172 9
138 155 7
103 27 7
128 39 10
39 151 8
117 120 7
166 79 9
71 81 9
98 82 10
6 82 10
93 11 8
95 151 9
120 136 9
123 24 9
49 189 7
17 82 8
3 61 8
91 2 10
141 21 8
165 163 9
174 79 9
132 162 9
121 151 7
133 1 9
144 173 8
19 13 8
57 65 7
85 176 7
131 113 9
58 82 8
41 151 9
184 17 8
126 28 9
12 88 10
32 21 7
169 161 10
77 47 10
109 46 10
50 25 10
140 179 8
172 28 9
29 152 10
136 122 7
170 55 8
187 41 7
188 82 7
1 88 9
92 82 8
106 81 9
107 41 7
129 27 8
13 11 9
31 189 9
59 166 9
114 28 8
20 11 10
75 166 7
21 151 7
68 79 7
73 141 10
47 81 9
43 66 7
127 161 10
160 90 9
72 79 10
110 189 9
156 79 9
44 79 10
67 55 8
11 193 10
171 11 9
51 81 7
79 193 9
28 193 8
180 82 8
102 83 7
45 169 9
40 151 7
163 193 9
24 193 10
196 168 9
94 79 8
23 142 7
130 11 9
119 82 9
145 197 9
116 193 10
46 81 7
27 36 7
167 22 7
34 196 10
55 193 8
143 9 9
15 111 7
97 79 9
142 81 9
199 4 7
176 28 9
56 162 9
52 81 9
151 193 10
84 151 7
69 95 7
5 189 9
112 151 9
178 164 10
78 79 8
149 79 8
185 11 7
61 81 7
125 66 7
36 111 8
161 28 10
147 163 7
108 9 8
26 77 7
175 11 9
30 151 7
42 180 7
113 28 10
87 157 7
70 193 9
191 39 10
122 82 9
37 157 10
134 121 8
152 81 10
4 81 8
115 119 8
146 24 7
158 189 7
186 11 7
139 79 8
53 79 10
173 47 10
14 11 10
25 28 9
48 151 10
177 76 9
35 151 9
33 82 9
