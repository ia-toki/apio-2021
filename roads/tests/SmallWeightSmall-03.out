064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
1692 1101 802 686 598 527 458 390 328 268 217 169 123 78 35 14 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
